src/components/Calendar/Calendar.tsx

```diff
@@ -121,6 +124,10 @@ const CalendarStyling = styled.div`
         background-color: ${(props) => props.theme.primary.background[1]};
         border-color: ${(props) => props.theme.primary.background[1]};
         color: ${(props) => props.theme.primary.foreground[1]};
+
+        &:hover {
+            opacity: 80%;
```

Comment:
I thought the themes change meant we shouldn't change opacity where possible, and just use a lighter colour?

Reply:
Sure. Changed to using `primary[2]`